packages/lib/pod/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@pcd/pod",
+  "version": "0.0.1",
+  "license": "MIT",
+  "main": "./dist/cjs/src/index.js",
+  "module": "./dist/esm/src/index.js",
+  "types": "./dist/types/src/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/types/src/index.d.ts",
+      "import": "./dist/esm/src/index.js",
+      "require": "./dist/cjs/src/index.js"
+    }
+  },
+  "files": [
+    "dist",
+    "./README.md",
+    "./LICENSE"
+  ],
+  "scripts": {
+    "lint": "eslint \"**/*.ts{,x}\"",
+    "build": "tsc -b tsconfig.cjs.json tsconfig.esm.json",
+    "typecheck": "yarn tsc --noEmit",
+    "prepublishOnly": "yarn clean && yarn build",
+    "test": "ts-mocha --config ../../../.mocharc.js --exit test/**/*.spec.ts",
+    "clean": "rm -rf dist node_modules *.tsbuildinfo"
+  },
+  "dependencies": {
+    "@pcd/util": "0.4.0",
+    "@zk-kit/baby-jubjub": "0.1.1",
+    "@zk-kit/eddsa-poseidon": "0.5.1",
+    "@zk-kit/imt": "2.0.0-beta.1",
+    "@zk-kit/utils": "0.1.0",
+    "js-sha256": "^0.10.1",
+    "json-bigint": "^1.0.0",
+    "poseidon-lite": "^0.2.0"
+  },
+  "devDependencies": {
+    "@pcd/eddsa-pcd": "0.5.0",
+    "@pcd/eslint-config-custom": "*",
```

Comment:
i think this is a mistake copy/pasted from elsewhere, but we should specify the specific version whenever we import, even if it's one of our own libs

Reply:
We use `'*'` for this and `@pcd/tsconfig` in most packages, which is what the template was based on. I'm changing that in https://github.com/proofcarryingdata/zupass/pull/1597